src/uu/date/src/date.rs

```diff
@@ -167,7 +185,7 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     };
 
     let date_source = if let Some(date) = matches.get_one::<String>(OPT_DATE) {
-        let ref_time = Local::now();
+        let ref_time = get_current_time(matches.get_flag(OPT_UNIVERSAL));
```

Comment:
I'd have a mild preference if you did this:

`let utc = matches.get_flag(OPT_UNIVERSAL);`

And used utc here and below when initializing `settings`.